lib/ws_servers/api/handlers/on_active_algo_orders_request.js

```diff
@@ -43,9 +42,10 @@ module.exports = async (server, ws, msg) => {
     return
   }
 
-  const markets = await Market.find([
-    ['p', '=', mode === 'paper' ? 1 : 0]
-  ])
+  const isPaper = mode === 'paper' ? 1 : 0
+
+  const marketValues = [...marketData.values()]
+  const markets = marketValues.filter(m => m.p === isPaper)
```

Comment:
`isPaper` is supposed to be a boolean type, but.. since `m.p` is a number, it can be converted to boolean
For example
```js
const isPaper = mode === 'paper'
/// 
marketValues.filter(m => Boolean(m.p) === isPaper)
```

Reply:
there are only two possible values for it .. `number to boolean` conversion seems like an extra step here